src/services/user.services.js

```diff
@@ -2,29 +2,24 @@ export const userService = {
     login
 };
 
-function login(email, password) {
+async function login(email, password) {
```

Comment:
Following the same ES6 nomenclature this should be: 
```suggestion
const login = async(email, password) {
```

Reply:
Excellent! Done in the PR of format improvement.